libs/authorizenet/src/effects/authorize-net.effects.ts

```diff
@@ -66,16 +73,33 @@ export class DaffAuthorizeNetEffects<T extends DaffAuthorizeNetTokenRequest = Da
 		map((actions: Actions[]) => new DaffAuthorizeNetUpdatePaymentFailure('The payment method has failed to update the cart.'))
 	)
 	
-	@Effect({ dispatch: false })
+	@Effect()
   loadAcceptJs$ : Observable<any> = this.actions$.pipe(
     ofType(DaffAuthorizeNetActionTypes.LoadAcceptJsAction),
-		tap((action: DaffLoadAcceptJs) => {
+		map((action: DaffLoadAcceptJs) => {
 			const acceptJsScript = document.createElement('script');
 			acceptJsScript.async = true;
 			acceptJsScript.setAttribute('type', 'text/javascript');
 			acceptJsScript.setAttribute('src', this.config.production ? ACCEPT_JS_PRODUCTION_URL : ACCEPT_JS_SANDBOX_URL);
 			acceptJsScript.setAttribute('charset', 'utf-8');
 			document.body.appendChild(acceptJsScript);
+			return new DaffCheckAcceptJs();
```

Comment:
I don't think we need this action, I think we can simply start the `checkAcceptJs$` when the `DaffLoadAcceptJs` triggers.

Reply:
This would have worked if I could've gotten backoff to work, but I don't think it makes sense without it.

Reply:
done